tools/rosbag_storage/include/rosbag/bag.h

```diff
@@ -569,7 +581,7 @@ void Bag::doWrite(std::string const& topic, ros::Time const& time, T const& msg,
 
         // Check if we want to stop this chunk
         uint32_t chunk_size = getChunkOffset();
-        logDebug("  curr_chunk_size=%d (threshold=%d)", chunk_size, chunk_threshold_);
+        CONSOLE_BRIDGE_logDebug("  curr_chunk_size=%d (threshold=%d)", chunk_size, chunk_threshold_);
```

Comment:
> It sounds like you want to just leave this be on Indigo; is that the case?

Yes, the version of `console_bridge` in Indigo doesn't have the namespaces macros. Therefore I don't think every higher level package needs to work around this upstream problem.

The patch could be backported to Kinetic if the `console_bridge` version on the platforms targeted by that support this.